tests/h/exceptions_test.py

```diff
@@ -20,3 +20,15 @@ def test_custom_status_code(self):
         exc = APIError('some message', status_code=418)
 
         assert exc.status_code == 418
+
+
+class TestClientUnauthorized(object):
+    def test_message(self):
+        exc = ClientUnauthorized()
+
+        assert 'credentials are invalid' in exc.message
```

Comment:
Because  `client credentials are invalid`, `Client credentials are invalid!` or `The supplied credentials are invalid.` would all be perfectly acceptable alternative error messages. Just trying to minimise the chances that small copy changes break the test.